src/server/src/modules/job.ts

```diff
@@ -240,11 +240,18 @@ export const updateJob = async (job: Job) => {
 };
 
 /**
- * @description Deletes an existing job from the database
+ * @description Deletes an existing job and relevant job applications from the database
  * @param {number} id - Id of job to delete
  * @returns Promise
  */
-export const deleteJob = (id: Job['id']) => {
+export const deleteJob = async (id: Job['id']) => {
+    await getConnection()
```

Comment:
is it possible to do `await JobApplication.delete({ jobId: id });` instead?

Reply:
Done.

Reply:
There are some conflicts